tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/services/ClientServiceDeclTest.java

```diff
@@ -0,0 +1,365 @@
+package org.ballerinalang.test.services;
```

Comment:
Shall we add the copyright headers?

Reply:
I will add

Reply:
Updated. e6adb7809bcf24f0faff2f7484b560a29cd03c36